src/scheduled/index.js

```diff
@@ -0,0 +1,82 @@
+let { env, maybeHydrate, readArc } = require('../helpers')
+let hydrate = require('@architect/hydrate')
+let runner = require('./_runner')
+let series = require('run-series')
+
+/**
+ * Looks for Scheduled Events and runs them as if they were cloudwatch events:
+ */
+module.exports = function createSchedule () {
+  let { arc } = readArc()
+
+  if (!arc.scheduled) {
+    return
+  }
+
+  let scheduled = {}
+  let schedulerBus
+
+
+  scheduled.start = function start (options, callback) {
+    let { all, update } = options
+
+    series([
+      function _env (callback) {
```

Comment:
gonna admit there's a lot of copy paste here
do you think it makes sense to have the hydrate stuff here or should i rather remove it?

Reply:
Yeah, we need it because all of our services may be started and stopped independently. In such an event, we need to ensure all functions have their dependencies.